src/models/generated/GroupReserve.jl

```diff
@@ -0,0 +1,78 @@
+#=
+This file was not auto-generated.
+=#
+"""
+    mutable struct StaticGroupReserve <: Service
+        name::String
+        available::Bool
+        requirement::Float64
+        ext::Dict{String, Any}
+        internal::InfrastructureSystemsInternal
+        contributing_services::Vector{<:PSY.Service}
+    end
+
+Data Structure for a group reserve product for system simulations.
+
+# Arguments
+- `name::String`
+- `available::Bool`
+- `requirement::Float64`: the static value of required reserves in system p.u., validation range: (0, nothing), action if invalid: error
+- `ext::Dict{String, Any}`
+- `internal::InfrastructureSystemsInternal`: power system internal reference, do not modify
+- `contributing_services::Vector{<:PSY.Service}`: Services that contribute for this requirement constraint
+"""
+mutable struct StaticGroupReserve <: Service
+    name::String
+    available::Bool
+    "the static value of required reserves in system p.u."
+    requirement::Float64
+    ext::Dict{String, Any}
+    "power system internal reference, do not modify"
+    internal::InfrastructureSystemsInternal
+    contributing_services::Vector{<:PSY.Service}
```

Comment:
> > Looks like the objective is to model a group of reserves. Only a few changes might be needed, probably we want to have specific methods for this object to limit StaticReserveGroups to only have other Reserves as contributing devices.
> 
> yes, exactly. The same way that `add_service!` [function](https://github.com/NREL-SIIP/PowerSystems.jl/blob/3ac11e010f25e074f2c18c55f90bdb97b23c4709/src/base.jl#L413) tests o see if the `contributing_devices` is filled with components of type `Device`, we could add a check at the constructor (or somewhere else if you prefer)

Actually, this is already enforced since the constructor only accepts `Vector{<:PSY.Service}`.